src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+
+    default String getPrintableString(){
+        return "this is the default interface string for Printable";
```

Comment:
we don't have to specify return value in interface, 
 simply `String getPrintableString();` will do